src/integrationTest/java/uk/gov/hmcts/reform/cpo/controllers/CasePaymentOrdersControllerIT.java

```diff
@@ -856,6 +860,7 @@ private void verifyDbCpoValues(UpdateCasePaymentOrderRequest request,
             assertEquals(CREATED_BY_IDAM_MOCK, updatedEntity.get().getCreatedBy());
             assertNotNull(updatedEntity.get().getCreatedTimestamp());
             assertTrue(previousCreatedTimestamp.isBefore(updatedEntity.get().getCreatedTimestamp()));
+            assertTrue(updatedEntity.get().isHistoryExists());
```

Comment:
Change to `assertEquals(HISTORY_EXISTS_UPDATED, updatedEntity.get().isHistoryExists());`  rather than assertTrue: to highlight that we expect it to be changed.